daprdocs/content/en/developing-applications/integrations/Azure/azure-authentication/howto-wif.md

```diff
@@ -0,0 +1,122 @@
+---
+type: docs
+title: "How to: Configure dapr to use workload identity federation on Azure"
+linkTitle: "How to: Configure dapr to use workload identity federation on Azure"
+weight: 20000
+description: "Learn how to configure dapr to use workload identity federation on Azure."
+---
+
+This guide will help you configure your Kubernetes cluster to run dapr with Azure workload identity federation.
+
+## What is it?
+
+[Workload identity federation](https://learn.microsoft.com/entra/workload-id/workload-identities-overview) 
+is a way for your applications to authenticate to Azure without having to store or manage credentials as part of 
+your releases.
+
+By using workload identity federation, any dapr components that target Azure will be able to authenticate transparently
+with no extra configuration! 🎉
+
+### How does it differ?
+
+Workload identity federation is one of a few ways Azure offers for your applications to gain access to Azure 
+resources.  Other options include:
+
+ - [Pod Managed Identities]({{< ref howto-mi.md >}}) - [Deprecated](https://learn.microsoft.com/azure/aks/use-azure-ad-pod-identity) method for authenticating applications at a pod level.
+ - [System and user assigned managed identities](https://learn.microsoft.com/azure/aks/use-managed-identity) - Less granular than workload identity federation.
+ - [Client ID and secret]({{ < ref howto-aad.md >}}) - Less recommended as it requires you to maintian and associate credentials at application level.
```

Comment:
I would move this into the overview, where it already says "Some Azure components offer alternative authentication methods, such as systems based on "shared keys" or "access tokens". Although these are valid and supported by Dapr, you should authenticate your Dapr components using Microsoft Entra ID whenever possible to take advantage of many benefits, including:" if you are saying the same thing here.